.scripts/configure.electron.ts

```
@@ -34,7 +34,7 @@ function updateWebConstant(setDesktopApp: boolean) {
 		const [from, to] = setDesktopApp ? [envCheck, hardcoded] : [hardcoded, envCheck];
 
 		if (!fileContent.includes(from)) {
-			throw new Error(`Expected content not found in ${filePath}`);
+			console.error(`Expected content not found in ${filePath}`);
```

@syns2191 maybe we can keep that `throw new Error(`Expected content not found in ${filePath}`);` here